lib/feature_flagger/model.rb

```diff
@@ -12,26 +12,30 @@ def self.included(base)
       base.extend ClassMethods
     end
 
-    def rollout?(feature_key)
+    def rollout?(*feature_key)
+      feature_key.flatten!
       resource_name = self.class.rollout_resource_name
       Feature.new(feature_key, resource_name).fetch!
       FeatureFlagger.control.rollout?(feature_key, id, resource_name)
     end
 
-    def release!(feature_key)
+    def release!(*feature_key)
+      feature_key.flatten!
       resource_name = self.class.rollout_resource_name
       Feature.new(feature_key, resource_name).fetch!
       FeatureFlagger.control.release!(feature_key, id, resource_name)
     end
 
-    def unrelease!(feature_key)
+    def unrelease!(*feature_key)
+      feature_key.flatten!
```

Comment:
The intention of this Pull Request is to allow the optional use of the array argument when checking for a rollout. 

This:

```ruby
unrelease!([:key1, :key2])
``` 

is very different from this 

```ruby
unrelease!(:key1, :key2)
```
The first example the arguments comes like this: `[[:key1, :key2]]`. The second example the arguments comes like this `[:key1, :key2]`.

The flatten here is to use the same behaviour for both method signatures.